src/tree.ts

```diff
@@ -103,6 +103,12 @@ export class LaunchTargetNode extends BaseNode {
             item.collapsibleState = vscode.TreeItemCollapsibleState.None;
             item.tooltip = localize("launch.target.currently.selected.for.debug.run.in.terminal",
                                     "The launch target currently selected for debug and run in terminal.\n{0}", this._toolTip);
+            // enablement in makefile.outline.setLaunchConfiguration is not

```

Comment:
Trying set this `TreeItem` to be disabled via [enablement](https://github.com/jdmchp/vscode-makefile-tools/blob/main/package.json#L177) in the `command`: `makefile.outline.setLaunchConfiguration`